core/network-libp2p/src/lib.rs

```diff
@@ -16,6 +16,9 @@
 
 //! Networking layer of Substrate.
 
+#[macro_use]
+extern crate slog_derive;
```

Comment:
Why do we need this?

Reply:
Because we want to derive SerdeValue for NetworkState

Reply:
Could be drop this line and include the macro where it is required, like it is done for the rest?

Reply:
sounds fair